scripts/glue/load-s3-to-db.py

```diff
@@ -0,0 +1,114 @@
+import os
+import sys
+import boto3
+import logging
+import psycopg2 
+import pandas as pd
+from io import BytesIO, StringIO
+from argparse import ArgumentParser
+from datetime import datetime, timedelta
+
+# Initialize logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
please create a singleTon class for a logging purposes!

import the instance of the class and use throughout, read [here](https://sourcemaking.com/design_patterns/singleton)

we can make a logger class that log meaningfully with multiple layers including trace stack for instance.